contracts/lib/AmountDeriverConstants.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.7;
```

Comment:
Why use 0.8.7 here?

Reply:
It is copied from ConsiderationConstants.sol let ask @0age should we change to >=0.8.13 or are there any specific cases? In my opinion, it should be >=0.8.13.

Reply:
![image](https://user-images.githubusercontent.com/4103490/172450919-33a7e0fc-51eb-4fff-b91e-bdd6a072fe49.png)
There are too many >=0.8.7 so, separate PR should be better.

Reply:
#416